System.Containers.Tasks/build/System.Containers.Tasks.targets

```diff
@@ -1,11 +1,18 @@
 <Project>
+    <Import Condition="'$(_IsTest)' == 'true'" Project="$(MSBuildThisFileDirectory)System.Containers.Tasks.props" />
```

Comment:
In theory we could build a test-specific targets file that imports both the props and targets. That's one way to avoid modifying the shipping targets for a test scenario

Reply:
this was pretty easy to do by glomming together the file contents and jamming that into a temp dir. Works for now :D